docs/requirements.txt

```diff
@@ -0,0 +1,19 @@
+GitPython==2.0.8
```

Comment:
Is it possible to share this with the envoy build somehow? And any other duplication? It's not a huge deal but will just be another thing to keep in sync.

Reply:
The plan is to suck in the main repo's `{requirements.txt, conf.py}` once we have the Bazel cross-linking story figured out. For the interim, it's easier to just fork with the understanding this is only temporary.